src/Sarif/Readers/PropertyBagConverter.cs

```diff
@@ -31,64 +31,24 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var propertyDictionary = new Dictionary<string, SerializedPropertyInfo>();
             foreach (string key in objectDictionary.Keys)
             {
-                JTokenType jTokenType = JTokenType.Undefined;
                 Type propertyType = objectDictionary[key].GetType();
 
-                if (propertyType == typeof(JObject))
```

Comment:
Again, we don't need the detailed logic to map between .NET types and JSON token types.